### PR TITLE
new devices

### PR DESCRIPTION
**Pull request by arteck**

### CODE REVIEW

File: devices.js [hunk 1]
```diff
@@ -1069,6 +1069,16 @@ const devices = [
...
         toZigbee: generic.light_onoff_brightness().toZigbee,
     },
+    {
+        zigbeeModel: ['RB 178 T'],
+        model: 'RB 178 T',
+        vendor: 'Innr',
+        description: 'E27 Bulb warm dimming',
+        supports: generic.light_onoff_brightness().supports,
+        fromZigbee: generic.light_onoff_brightness().fromZigbee,
+        toZigbee: generic.light_onoff_brightness().toZigbee,
```
Comment (Koenkk) [inline devices.js]: It seems that this device also supports color temperature, is that right? (https://www.amazon.co.uk/tunable-white-RB-178-pack/dp/B075QM4YLX)

**Comment by Koenkk** [comment]
@arteck did you see my comment?

**Comment by arteck** [comment]
ohh sry yes.. you're right

File: devices.js [hunk 2]
```diff
@@ -1882,6 +1910,16 @@ const devices = [
         fromZigbee: [fz.heiman_water_leak],
         toZigbee: [],
     },
+    
+    {
+        zigbeeModel: ['HOMA2023'],
+        model: 'ZB-CC-40',
```
Comment (Koenkk) [inline devices.js]: Could you provide a link to this device (to be added to the documentation)?

Comment (arteck) [reply devices.js]: its a noname LED driver from Aliexpress.. look like a Gledopto driver.
one of our users has bought it

Comment (Koenkk) [reply devices.js]: Can you give the aliexpress link?

**Pull request merged**